packages/perseus-score/src/score.test.ts

```diff
@@ -0,0 +1,590 @@
+import {flattenScores, scorePerseusItem, scoreWidgetsFunctional} from "./score";
+import {
+    getExpressionWidget,
+    getLegacyExpressionWidget,
+    getTestDropdownWidget,
+} from "./util/test-helpers";
+
+import type {UserInputMap} from "./validation.types";
+import type {
+    DropdownWidget,
+    PerseusWidgetsMap,
+} from "@khanacademy/perseus-core";
+
+describe("flattenScores", () => {
```

Comment:
Most of this is copied as-is except for the removal of `strings` and the refactor I pointed out.